src/docs/development/tools/vs-code.md

```diff
@@ -232,6 +232,9 @@ The Flutter extension includes the following snippets:
 
 You can also define custom snippets by executing
 **Configure User Snippets** from the [Command Palette][].
+### Converting Stateful Widget to Stateless Widget
+Converting stateless to statefull can be used to spead up entering typical code structure.
```

Comment:
- Converting **StatelessWidget to StatefulWidget**..
- spead -> speed
- Apart from this, the meaning of this sentence is ambiguous.